witchcraft/internal/tcpjson/async_writer.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2021 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tcpjson
+
+import (
+	"io"
+	"log"
+
+	"github.com/palantir/pkg/metrics"
+	werror "github.com/palantir/witchcraft-go-error"
+)
+
+const (
+	asyncWriterBufferCap      = 1000
```

Comment:
Is there any context around this value or is it arbitrary? Also is this bytes or number of messages or something else?